src/chess/Command.java

```diff
@@ -18,6 +18,12 @@
   static Command parse(String str) {
     switch (str.toLowerCase()) {
       case "help":
+        System.out.println("* type 'help' for help");
+        System.out.println("* type 'board' to see the board again");
+        System.out.println("* type 'resign' to resign");
+        System.out.println("* type 'moves' to list all possible moves");
+        System.out.println("* type a square (e.g. b1, e2) to list possible moves for that square");
+        System.out.println("* type a UCI (e.g. b1c3, e7e8q) to make a move");
```

Comment:
You don't actually need to print several times
It's better to print once to better performance

I'm not sure which form of codes do you prefer, but you can write like below

```java
System.out.println("* type 'help' for help"
    + "* type 'board' to see the board again"
    + "* type 'resign' to resign"
    + "* type 'moves' to list all possible moves"
    + "* type a square (e.g. b1, e2) to list possible moves for that square"
    + "* type a UCI (e.g. b1c3, e7e8q) to make a move");
```

```java
System.out.println(""
    + "* type 'help' for help"
    + "* type 'board' to see the board again"
    + "* type 'resign' to resign"
    + "* type 'moves' to list all possible moves"
    + "* type a square (e.g. b1, e2) to list possible moves for that square"
    + "* type a UCI (e.g. b1c3, e7e8q) to make a move");
```

```java
System.out.println(""
    + "* type 'help' for help"
    + "* type 'board' to see the board again"
    + "* type 'resign' to resign"
    + "* type 'moves' to list all possible moves"
    + "* type a square (e.g. b1, e2) to list possible moves for that square"
    + "* type a UCI (e.g. b1c3, e7e8q) to make a move"
);
```